src/components/chat/comment-textbox.tsx

```diff
@@ -40,29 +49,50 @@ export const CommentTextBox: React.FC<IProps> = ({ activeNavTab, numPostedCommen
   };
 
   const handlePostComment = () => {
-    onPostComment?.(commentText);
-    setCommentTextAreaHeight(minTextAreaHeight);
-    setCommentAdded(false);
-    setCommentText("");
-  };
-
-  const handleEscKey = (event: React.KeyboardEvent) => {
-    if(event.key === "Escape") {
+    // do not send post if text area is empty, only has spaces or new lines
+    const content = commentText.trim().replace(/\s\s+/g, " ");
```

Comment:
I'm not sure what to suggest as an alternative, but it seems to me that this will do the wrong thing in some cases, e.g. `"line 1 \nline 2"`, where the space-newline combination will get converted into a space. This will also prevent adding spaces at the beginning of a line for indentation. What about something like:
```suggestion
    const content = commentText.trim().replace(/\s+\n/g, "\n");
```
where the idea would be to eliminate white-space at the beginning/end of the string (via the call to `trim()`) and at the end of each line (via the regex) but preserve interior white-space.